tools/IncrementVersion.ps1

```diff
@@ -0,0 +1,58 @@
+$versionPath = "$PSScriptRoot\CustomMSBuild\Versioning.props"
+
+$lastReleaseCommit = git log -n 1 --pretty=format:%H -- $versionPath
+Write-Host "The last release's commit was $lastReleaseCommit"
+
+$changedFiles = git diff --name-only HEAD $lastReleaseCommit
+
+$apiChanges = $false
+ForEach ($line in $($changedFiles -split "`r`n"))
+{
+  $fileName = [System.IO.Path]::GetFileName($line)
```

Comment:
why can't compare the content between Shipped and unShipped?

Reply:
I don't want this script to need to know all of the projects that we release in order to do the task of incrementing version numbers. I suppose that it could navigate the directory structure to find the files with these names, but generally speaking there are far fewer files changed in our commits between releases than there are files in our repo, so I'd rather use this.